src/main/resources/application.yml

```diff
@@ -0,0 +1,160 @@
+server:
+  port: 9000
+  contextPath: 
+
+# use this as the security key for encryption
+security:
+  user:
+    password: 81d56928-891a-41b2-8dd2-132593e6d057
+
+spring:
+  main:
+    allow-bean-definition-overriding: true
+    allow-circular-references: true
+  template:
+    cache: false
+  http:
+    multipart:
+      maxFileSize: 20MB
+      maxRequestSize: 20MB
+  sql:
+    init:
+      platform: h2
+      # platform: postgresql
+      # platform: mysql
+  datasource:
+    # H2
+    url: jdbc:h2:mem:AZ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    driverClassName: org.h2.Driver
+    # PostgreSQL
+    # url: jdbc:postgresql://localhost:5432/cap
+    # driverClassName: org.postgresql.Driver
+    # MySQL
+    # url: jdbc:mysql://localhost/cap
+    # driverClassName: com.mysql.jdbc.Driver
+    jpa:
+      database-platform: org.hibernate.dialect.H2Dialect
```

Comment:
This is incorrect.

The `spring.datasource.jpa.dabase-platform` needs to be changed to `spring.jpa.database-platform`.

There is a `jpa` around line 46 below that this should be moved to.

see: [spring.jpa.database-platform](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.data.spring.jpa.database-platform)